benchmark/scenarios/_rate_measurer.py

```diff
@@ -30,23 +36,25 @@ def request_sent(self):
         """
         self._sent += 1
 
-    def response_received(self, ignored):
+    def response_received(self, calltime):
         """
         Increase the number of received requests.
 
-        :param ignored: The result of a callback. This parameter is
-            not used.
+        :param float calltime: Time to perform call
         """
         self._received += 1
+        calltime = int(calltime)
```

Comment:
:question: I saw your comment in the cover letter. Could we perhaps store these as milliseconds for increased granularity? It'd be nice to aim high.